activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/process/IntermediateFlowNodeIncomingOutgoingFlowValidator.java

```diff
@@ -26,12 +26,14 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static java.lang.String.format;
+
 public class IntermediateFlowNodeIncomingOutgoingFlowValidator implements FlowNodeFlowsValidator {
 
     public static final String NO_INCOMING_FLOW_PROBLEM = "Intermediate Flow node has no incoming flow";
-    public static final String NO_INCOMING_FLOW_PROBLEM_DESCRIPTION = "Intermediate Flow node has to have an incoming flow";
+    public static final String NO_INCOMING_FLOW_PROBLEM_DESCRIPTION = "Intermediate Flow node name: '%s', id: '%s' has to have an incoming flow";
```

Comment:
Just a small readability improvement.
 
Can the description just be `name: '%s', id: '%s' has to have an incoming flow`? It would be easier to read and `Intermediate Flow node has to have an incoming flow` is already mentioned in the title.
 Same applies for outgoing `name: '%s', id: '%s' has to have an outgoing flow`. And maybe we want to do the same for start/end events  and sequence flows.

Reply:
I think `mergify` merged the PR before I can fix it.  Just created another #245